src/routes/(protected)/settings/your-apps/new/+page.svelte

```diff
@@ -2,11 +2,33 @@
 	import { superForm } from 'sveltekit-superforms/client';
     export let data;
     const {form, errors, constraints} = superForm(data.form);
+    function urlBoxes(callingObject) {
+        if (callingObject.name === "redirectUri" || callingObject.name === "supportServer") {
+            if (callingObject.name === "redirectUri") {
+                $form.redirectUri = callingObject.value.toLowerCase()
+            } else if (callingObject.name === "supportServer") {
+                $form.supportServer = callingObject.value.toLowerCase()
+            }
+            if (callingObject.value.toLowerCase().startsWith("https://")) {
+                callingObject.value = callingObject.value.split("https://")[1]
+                document.getElementsByName(callingObject.name)[0].innerText = "https://"
+            } else if (callingObject.value.toLowerCase().startsWith("http://")) {
+                document.getElementsByName(callingObject.name)[0].innerText = "http://"
+                callingObject.value = callingObject.value.split("http://")[1]
+            } 
+        } else if (callingObject.name === "vanityCode" && callingObject.value.includes("cardboard.ink/a/")){
+            $form.vanityCode = callingObject.value.split("cardboard.ink/a/")[1]
+            callingObject.value = callingObject.value.split("cardboard.ink/a/")[1]
+        }
+    }
```

Comment:
instead of updating dom values, you can just update the js variable like form.name = "newValueHere"